aml-development/frontend
# feat(storefront): allow custom ordering of storefront sections

AMLOS-763

**Description**
AML 2.0 has a Sort dropdown for the Most Popular section on storefront. Sort would be beneficial for most Storefront See More pages.

**Acceptance Criteria**
- The use can sort See More pages (Newest, Title, Rating)

**How to test code**
Pull this branch
Pull `see_more_storefront` from `ozp-backend` if https://github.com/aml-development/ozp-backend/pull/455 is not merged

- Verify that all _See More_ pages on the storefront, except for _Updates_, has a sort drop-down.
- Verify that changing the sort option updates the order of the listings appropriately.
- Verify that clicking _Back_ to go to the storefront again will, if necessary, reload the listings in the default sort order.
